package.json

```diff
@@ -286,7 +286,7 @@
 		"lint-fix": "eslint src --ext ts --fix && npm run lint-fix --prefix webview-ui",
 		"lint-fix-local": "eslint -c .eslintrc.local.json src --ext ts --fix && npm run lint-fix --prefix webview-ui",
 		"package": "npm run build:webview && npm run check-types && npm run lint && node esbuild.js --production",
-		"pretest": "npm run compile && npm run compile:integration",
```

Comment:
This was an unnecessary step for just running `jest`, and we already run it as part of `test:integration`.